ethcore/src/miner/miner.rs

```diff
@@ -307,6 +307,24 @@ impl Miner {
 		!have_work
 	}
 
+	fn add_transactions_to_queue(&self, chain: &MiningBlockChainClient, transactions: Vec<SignedTransaction>, origin: TransactionOrigin) ->
+		Vec<Result<TransactionImportResult, Error>> {
+
+		let fetch_account = |a: &Address| AccountDetails {
+			nonce: chain.latest_nonce(a),
+			balance: chain.latest_balance(a),
+		};
+
+
+		let results: Vec<Result<TransactionImportResult, Error>> = {
```

Comment:
You can get rid of closure and return right away. The type and assignment to `results` would be unnecessary